app/models/dscc_number.rb

```diff
@@ -0,0 +1,29 @@
+class DsccNumber < ActiveRecord::Base
+
+  def self.current_monthly_number
+    where(year_and_month: current_year_and_month).order('number desc').limit(1).pluck(:number).first || 0
+  end
+
+  def self.next_monthly_number
+    current_monthly_number + 1
+  end
+
+  def self.current_year_and_month
+    Time.current.to_date.beginning_of_month
```

Comment:
Scratch that - we are storing it. :smile:

But why are we storing it - does `created_at` not give us what we need?